tests/common/unittest_common.cc

```diff
@@ -1664,6 +1664,43 @@ TEST (confCustom, envStr01)
   }
 }
 
+/**
+ * @brief Test for extra configuration path
+ */
+TEST (confCustom, checkExtraConfPath_p)
+{
+  gchar *fullpath = g_build_path ("/", g_get_tmp_dir (), "nns-tizen-XXXXXX", NULL);
+  gchar *dir = g_mkdtemp (fullpath);
+  gchar *filename = g_build_path ("/", dir, "nnstreamer.ini", NULL);
+  const gchar *extra_conf = "/opt/usr/vd/product.ini";
+  const gchar *base_confenv;
+  gchar *confenv;
+  FILE *fp = g_fopen (filename, "w");
+  ASSERT_TRUE (fp != NULL);
+
+  base_confenv = g_getenv ("NNSTREAMER_CONF");
+  confenv = (base_confenv != NULL) ? g_strdup (base_confenv) : NULL;
```

Comment:
IMO base_confenv is unnecessary,
`confenv = g_strdup (g_getenv ("NNSTREAMER_CONF"));`